docs/user/advanced_features.rst

```diff
@@ -868,6 +868,316 @@ Constraint and PTO Features
 .. include:: /_include/pto.rst
 
 
+
+.. _user-advanced-features-control:
+
+Controller Features
+-------------------
+
+Controllers for wave energy conversion can be used to determine the force 
+applied by the power take off onto the device. WEC-Sim's controller 
+features support the modeling of both simple passive and reactive controllers 
+as well as more complex methods such as model predictive control. 
+
+The files for the controller tutorials described in this section can be found in 
+the **Controls** examples on the `WEC-Sim Applications repository 
+<https://github.com/WEC-Sim/WEC-Sim_Applications>`_ . The controller examples 
+are not comprehensive, but provide a reference for user to implement their 
+own controls.
+
+	+--------------------------------+-------------------------------------------+
+	|   **Controller Application**   |               **Description**             |                
+	+--------------------------------+-------------------------------------------+
+	|   Passive (P)  	         | RM3 Float with proportional control       |
+	+--------------------------------+-------------------------------------------+
+	|   Reactive (PI)                | Float with proportional-integral control  |
+	+--------------------------------+-------------------------------------------+
+	|   Latching		         | Float with latching control		     |
+	+--------------------------------+-------------------------------------------+
+	|   Declutching   		 | Float with declutching control            |
+	+--------------------------------+-------------------------------------------+
+	|   Model Predictive Control   	 | Float with model predictive control       |
+	+--------------------------------+-------------------------------------------+
+
+
+
+Examples: RM3 Float with Various Controllers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section explains the controller examples found within the WEC-Sim 
+Applications repository.
+
+First, it is important to understand the concept of complex conjugate control.
+Complex conjugate control, as applied to wave energy conversion, 
+can be used to understand optimal control. For a complex conjugate controller, 
+the impedance of the controller is designed to match the admittance of the 
+device which is equal to the complex conjugate of the impedance. Hence, it is 
+also known as impedance matching and is a common practice within electrical 
+engineering. Complex conjugate control is not a completely realizable control 
+method due to its acausality. which means it requires exact knowledge of 
+future wave conditions. Still, complex conjugate control presents a reference 
+for the implementation of optimal control. The WEC impedance can be modeled by 
+the following equation and can be used to formulate optimal control 
+implementation:
+
+.. math::
```

Comment:
This equation is appropriate for a single oscillating body, but is not generalizable to a multiple body system. As such, I am still in favor of an example case as this class is not going to be fully generalizable to more complex WECs that we see users researching through issues.